patch-target.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+set -euo pipefail
+
+function usage() {
+    cat <<USAGE
+Usage: ${0} <source-dir> <target-dir> <commit-message>
+
+Commits a patch to <target-dir> with the relevant changes from
+<source-dir> and the given <commit-message>, and outputs a JSON
+document with the files that were changed.
+
+Environment variables
+
+\$UPDATE_BINARY: Set this to the path of the update-from-template
+  executable.  Defaults to '/update-from-template'.  Currently set to
+  '${update_binary}'.
+
+\$UPDATE_CONFIG_FILE: Set this to the path of the config file for
+  the update-from-template executable.  Defaults to '/.config.yaml'.
+  Currently set to '${update_config_file}'.
+
+USAGE
+}
+
+src_dir=${1:-}
+dest_dir=${2:-}
+commit_msg=${3:-}
+update_binary=${UPDATE_BINARY:-/update-from-template}
+update_config_file=${UPDATE_CONFIG_FILE:-/.config.yaml}
```

Comment:
This should be `/update-from-template.yaml` rather than `/.config.yaml`. It is copied to the root level in the Dockerfile from the project's directory `scripts/config/update-from-template.yaml`.